src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.constant.Color;
+import core.basesyntax.producers.ColorProducer;
+
+public abstract class Figure implements Drawable {
+    public abstract double getArea();
```

Comment:
move method to an interface - that's behavior, interfaces are used to represent behavior

Reply:
done!